packages/core/lib/settings/config.dart

```diff
@@ -47,13 +47,19 @@ flutter_gen:
     enabled: true
     package_parameter_enabled: false
     style: dot-delimiter
+    outputs:
+      class_name: Assets
     
   fonts:
     enabled: true
+    outputs:
+      class_name: FontFamily
```

Comment:
Same as above comment becoz you checked here https://github.com/FlutterGen/flutter_gen/pull/280/files#diff-74b9255cc2b20267c715b8162a5fcd4e4d55b3b0fb333a5bc23b52deb5d207e4R22